src/elm/Main.elm

```diff
@@ -112,6 +113,7 @@ subscriptions model =
         , InView.subscriptions InViewMsg model.inView
         , onScroll OnScroll
         , Pile.subscriptions model.piles |> Sub.map Piles
+        , Browser.Events.onResize (\newWidth newHeight -> OnResize ( toFloat newHeight, toFloat newHeight ))
```

Comment:
Good idea - always think it's mostly devs who change size of window after load, but probably many people do in different situations.